multimedia/scrcpy/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           meson 1.0
+
+github.setup        Genymobile scrcpy 1.17 v
+github.tarball_from releases
+revision            0
+
+categories          multimedia
+platforms           darwin
+license             Apache-2
+maintainers         {bochtler.io:macports @MarcelBochtler} \
+                    openmaintainer
+description         Display and control your Android device
+long_description    This application provides display and control of Android \
+                    devices connected on USB (or over TCP/IP). It does not \
+                    require any root access.
+
+distfiles-append    scrcpy-server-v${version}
+
+checksums           ${distname}${extract.suffix} \
+                    rmd160  b0909f342a61e6001fe1643df98f38d0dcc99641 \
+                    sha256  b3f5490970dbef6f50725b4d46ba07baed9c7eb19696e71029257a01044c6664 \
+                    size    266684 \
+                    scrcpy-server-v${version} \
+                    rmd160  59bc2a451016f5c5f11a22643a441a19e3a78b9b \
+                    sha256  11b5ad2d1bc9b9730fb7254a78efd71a8ff46b1938ff468e47a21b653a1b6725 \
+                    size    34930
+
+depends_build-append \
+                    port:pkgconfig
+
+depends_lib-append  path:lib/libavcodec.dylib:ffmpeg \
+                    port:libsdl2
+
+depends_run-append  bin:adb:android-platform-tools
```

Comment:
There is no devel version of android-platform-tools available in Macports. So make this a port dependency. 

`port:android-platform-tools`